gradle-baseline-java/src/test/groovy/com/palantir/baseline/BaselineIntegrationTest.groovy

```diff
@@ -43,6 +43,6 @@ class BaselineIntegrationTest extends AbstractPluginTest {
         with().withArguments('-s').withGradleVersion(gradleVersion).build()
 
         where:
-        gradleVersion << ['4.10.2', '5.0-milestone-1']
+        gradleVersion << ['5.0']
```

Comment:
could we add `6.0-20190904072820+0000` here?

Reply:
sure